src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
```

Comment:
Probably make this self-by-value too, since it reduces the number of allocations you need to do. If the caller needs to keep the original `Rational` around, they can clone themselves.